src/stage1/recurse_fwd.jl

```diff
@@ -35,8 +35,10 @@ function perform_fwd_transform(world::UInt, source::LineNumberNode,
             Core.svec(:ff, :args), Core.svec(), :(∂☆passthrough(args)))
     end
 
-    # Check if we have an rrule for this function
+    # Check if we have an frule for this function
     sig = Tuple{map(π, args)...}
+    Core.println("sig: ", sig)
+    Core.println("world: ", world)
     mthds = Base._methods_by_ftype(sig, -1, world)
```

Comment:
It is currently broken due to it not being able to find the methods for any varadic functions and idk what changed there.
this line always returns `nothing`